File: util/getLatestCTREVersion.java
import java.util.ArrayList;

public class getLatestCTREVersion {

    public static void main(String[] args) {
        String val = "";
        String search = args[args.length - 1];
        args[args.length - 1] = "";
        for(int i = 0; i < args.length; i++) {
            if(args[i].contains(search)) val = args[i];
        }

        // " v5.0.3.3.zip" is a hypothetical versioning. They're all the same length
        int startIndex = val.indexOf(search);
        int endIndex = startIndex + "v5.0.3.3.zip".length();
        System.out.println(val.substring(startIndex, endIndex));
    }

}

File: util/getLatestCTREVersion.py
import sys

search = sys.argv[-1]
val = ''
for i in sys.argv[:-1]:
    if search in i:
        val = i

# " v5.0.3.3.zip" is a hypothetical versioning. They're all the same length
startindex = val.index(search)
endindex = startindex + len(" v5.0.3.3.zip")
print(val[startindex:endindex])
